src/main/java/net/sf/jabref/gui/dbproperties/DatabasePropertiesDialog.java

```diff
@@ -79,6 +80,18 @@ public void setPanel(BasePanel panel) {
         this.metaData = panel.getBibDatabaseContext().getMetaData();
     }
 
+    public void updateEnableStatus() {
+        DatabaseLocation location = panel.getBibDatabaseContext().getLocation();
+        boolean isNotShared = (location != DatabaseLocation.SHARED);
+        encoding.setEnabled(isNotShared);
+        fileDir.setEnabled(isNotShared);
```

Comment:
`fileDir` is the directory, where the local PDFs are stored. This has to be available on shared databases, too. So please remove this and the next line.